lib/src/main/java/ECSE428Project/controller/LogoutController.java

```diff
@@ -0,0 +1,37 @@
+package ECSE428Project.controller;
+
+import ECSE428Project.dto.AccountDto;
+import ECSE428Project.service.LogoutService;
+//import ECSE428Project.controller.LoginController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin(origins = "*")
+@RestController
+public class LogoutController {
+
+    @Autowired
+    private LogoutService logoutService;
+
+    //Logging out
+	@PostMapping(value = { "/logout", "/logout/" })
+	public AccountDto logoutProfile(@RequestParam("email") String email) {
+		if (!logoutService.profileLogout(email).isLoggedIn()) {
+			return convertToDTO(email, false);
+		} else {
+			throw new IllegalArgumentException("Logout failed.");
+		}
+	}
+
+
+    //This DTO method can be reused across multiple controller classes
```

Comment:
I created a public method AccountDto convertToDto(Account account) in the DataTransferObject class.
You could either change your method a bit to use that or you could put your convertToDto in that class so that it can be reused